clustering/ejb/client/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
```

Comment:
2010? I can't believe it but my wristwatch says it's 2021 already!

Reply:
:1st_place_medal: